### PR TITLE
[QC-953] mergers can receive labels through config

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

**Comment by Barthelemy** [comment]
@knopers8 Could you review when you got a moment ?

**Comment by ktf** [comment]
Can we go forward with this? As I understand it block the deployment of the expendable tasks.

**Comment by Barthelemy** [comment]
@ktf  could you have a look and approve ? Piotr is on holidays

File: Utilities/Mergers/src/MergerBuilder.cxx [hunk 1]
```diff
@@ -116,10 +116,7 @@ framework::DataProcessorSpec MergerBuilder::buildSpec()
 
   merger.inputs.push_back({"timer-publish", "TMR", mergerDataDescription(mName), mergerSubSpec(mLayer, mId), framework::Lifetime::Timer, timerSpecs(timers)});
   merger.labels.push_back(mergerLabel());
-  if (mConfig.expendable) {
-    framework::DataProcessorLabel expendableLabel = {"expendable"};
-    merger.labels.emplace_back(expendableLabel);
-  }
+  merger.labels.insert(merger.labels.end(), mConfig.labels.begin(), mConfig.labels.end());
```
Comment (ktf) [inline Utilities/Mergers/src/MergerBuilder.cxx:119]: maybe you should also do `std::unique`?

Comment (Barthelemy) [reply Utilities/Mergers/src/MergerBuilder.cxx:119]: I did not realize it was a vector. Ok, done.

**Comment by Barthelemy** [comment]
Should be good now

**Comment by Barthelemy** [comment]
@ktf perhaps you could merge it ?

**Pull request merged**